src/scss/css.scss

```diff
@@ -24,6 +24,21 @@
     bottom: 0 !important
 }
 
+
+#drop {
```

Comment:
I would like to use a class here instead of modifying an element based on an id, since ids should be unique and we have multiple tooltips. We could introduce some semantics by naming that class something like 'carbonara-mobile-tooltip'.

Reply:
Fixed!